tools/util/include/cutlass/util/reference/device/gemm_complex.h

```diff
@@ -246,38 +242,23 @@ using syclcompat::dim3;
 
   if (grid.y <= std::numeric_limits<uint16_t>::max()) {
 #if defined(CUTLASS_ENABLE_SYCL)
-
+  syclcompat::dim3 sycl_grid(grid.x, grid.y, grid.z);
+  syclcompat::dim3 sycl_block(block.x, block.y, block.z);
```

Comment:
What's the point of these two lines? Couldn't you use dimBlock/dimGrid directly?

Reply:
There is no conversion between `dim3` and `syclcompat::dim3`